pkgs/development/gnuradio-modules/osmosdr/default.nix

```diff
@@ -39,6 +39,8 @@ in mkDerivation {
   inherit version src;
   disabledForGRafter = "3.9";
 
+  outputs = [ "out" "dev" ];
```

Comment:
With this, I'm not sure it won't break the setups of some of our `gnuradio/wrapper.nix` users..